onset_maproom/calc.py

```diff
@@ -268,6 +268,23 @@ def seasonal_sum(
     summed_seasons = xr.merge([seasonal_data, seasons_ends])
     return summed_seasons
 
+def probExceed(onsetMD, earlyStart):
+    onsetMD_df = onsetMD
```

Comment:
Should we try to make a generic probExceed function that takes an xarray and the coord against which the probExceed is to be computed against, and probably like in Ingrid, the range and step to apply to the binning?

In that case the specifics of the onset date case should be taken away (e.g. earlyStart) and dealt with outside of the function; and should try to use xarray objects.

For instance, I wonder if we could use [quantile](https://xarray.pydata.org/en/stable/generated/xarray.DataArray.quantile.html) (unless there is something more directly appropriate -- I still am not too good at searching existing functions). Then the exercise is mostly to construct q with the right range and step and then do some manips to get percent rather than quantiles, and maybe calculate the complementary (1-x).

Reply:
I think it's ok to focus on the specific case at hand at first. We can generalize later.

Reply:
That is definitely something I can work on at some point once I have this specific case finished.